src/test/java/com/galvanize/speedway/integrationtest/DriverIT.java

```diff
@@ -30,15 +37,41 @@ public void getDriversTest() throws Exception{
       this.mockMvc.perform(get("/speedway/drivers"))
          .andExpect(status().isOk())
          .andExpect(jsonPath("length()").value(0))
-//         .andDo(document("GetDrivers",responseFields(
-//            fieldWithPath("[0].firstName").description("First Name of driver"),
-//            fieldWithPath("[0].lastName").description("Last Name of driver"),
-//            fieldWithPath("[0].age").description("Age of driver"),
-//            fieldWithPath("[0].nickName").description("Nick Name of driver"),
-//            fieldWithPath("[0].wins").description("Wins of driver"),
-//            fieldWithPath("[0].losses").description("Losses of driver"),
-//             fieldWithPath("[0].cars").description("Cars the driver has driven")
-//         )))
          ;
    }
+
+   @Test
+   public void postDriversTest() throws Exception{
+
+      Driver driver = new Driver();
+      driver.setFirstName("Test First Name");
+      driver.setLastName("Test lastname");
+      driver.setLosses((short) 2);
+      driver.setWins((short) 5);
+      driver.setAge((short) 25);
+      driver.setNickName("Test Nickname");
+
+
+      RequestBuilder rq = post("/speedway/drivers")
+              .content(objectMapper.writeValueAsString(driver))
+              .contentType(MediaType.APPLICATION_JSON);
+
+      mockMvc.perform(rq)
+              .andExpect(status().isCreated())
+              .andDo(print());
```

Comment:
Did you want to document how `POST`s should be formatted?

Reply:
I will add it